wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -56,7 +56,7 @@ class WirePlugin : Plugin<Project> {
     project.configurations.create("protoPath")
       .also {
         it.isCanBeConsumed = false
-        it.isTransitive = false
+        it.isTransitive = true
```

Comment:
Won't this pull-in all dependencies? like if we depend on `my.service:client:version`, we could get the `.proto` files but as well, all `.class`, etc?